src/main/java/org/mqttbee/mqtt/MqttClientData.java

```diff
@@ -130,6 +141,11 @@ public boolean isConnected() {
         return connected.get();
     }
 
+    @Override
```

Comment:
Ordering of methods should be:
...
getServerPort
usesSsl
getSslConfig
usesWebsockets
getWebsocketConfig
isConnecting
...

Reply:
DONE